server/src/main/java/org/candlepin/dto/api/v1/ActivationKeyContentOverrideDTO.java

```diff
@@ -0,0 +1,152 @@
+        this.populate(source);
+    }
+
+    /**
+     * Fetches the activation key that's overriding the content. If the activation key has not been
+     * set, this method returns null.
+     *
+     * @return
+     *  The activation key overriding the content, or null if the activation key has not been set
+     */
+    @JsonIgnore
+    public ActivationKeyDTO getActivationKey() {
+        return this.activationKey;
+    }
+
+    /**
+     * Sets or clears the activation key that's overriding the content. If the provided key is null,
+     * any key currently set will be cleared.
+     *
+     * @param activationKey
+     *  The activation key that's overriding the content, or null to clear the key
+     *
+     * @return
+     *  A reference to this DTO
+     */
+    @JsonProperty("key")
+    public ActivationKeyContentOverrideDTO setActivationKey(ActivationKeyDTO activationKey) {
+        this.activationKey = activationKey;
+        return this;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String toString() {
+        ActivationKeyDTO key = this.getActivationKey();
+
+        return String.format("ActivationKeyContentOverrideDTO [key: %s, content: %s, name: %s, value: %s]",
+            this.getContentLabel(), this.getName(), this.getValue(), (key != null ? key.getId() : null));
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == this) {
+            return true;
+        }
+
+        if (obj instanceof ActivationKeyContentOverrideDTO && super.equals(obj)) {
+            ActivationKeyContentOverrideDTO that = (ActivationKeyContentOverrideDTO) obj;
+
+            ActivationKeyDTO thisKey = this.getActivationKey();
+            ActivationKeyDTO thatKey = that.getActivationKey();
+
+            String thisKeyId = thisKey != null ? thisKey.getId() : null;
+            String thatKeyId = thatKey != null ? thatKey.getId() : null;
+
+            return thisKeyId != null ? thisKeyId.equals(thatKeyId) : thatKeyId == null;
+        }
+
+        return false;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public int hashCode() {
+        ActivationKeyDTO key = this.getActivationKey();
+
+        HashCodeBuilder builder = new HashCodeBuilder(37, 7)
+            .append(super.hashCode())
+            .append(key != null ? key.getId() : null);
+
+        return builder.toHashCode();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ActivationKeyContentOverrideDTO clone() {
+        ActivationKeyContentOverrideDTO copy = super.clone();
+
+        ActivationKeyDTO key = this.getActivationKey();
+        copy.setActivationKey(key != null ? key.clone() : null);
+
+        return copy;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ActivationKeyContentOverrideDTO populate(ActivationKeyContentOverrideDTO source) {
+        super.populate(source);
+
```

Comment:
Need to add `this.setActivationKey(source.getActivationKey());` here.

Reply:
Fixed.